### ISSUE TITLE
why are some python files executable?

### ISSUE THREAD

**Issue by timotheecour** [Varriount/NimLime#119]
for eg:
settings.py is not executable
but configuration.py is (and a number of others are also)
```
ls core
total 12K
drwxr-xr-x 8 timothee  256 Nov  5 16:05 commands/
drwxr-xr-x 8 timothee  256 Nov  5 16:05 utils/
-rwxr-xr-x 1 timothee   15 Nov  5 16:05 __init__.py
-rwxr-xr-x 1 timothee 1.1K Nov  5 16:05 configuration.py
-rw-r--r-- 1 timothee 2.5K Nov  5 16:05 settings.py
```

**Comment by Varriount** [comment]
No idea. I'll fix those.

**Comment by timotheecour** [comment]
thanks! but in general why not keep the issue open until the fix is in `master` ?  (github will automatically close it if commit msg contains "fixes #12345" as I'm sure you know)
